build.gradle

```diff
@@ -25,21 +25,21 @@ ext {
     slf4jVersion = '1.7.7'
     log4jVersion = '1.2.17'
     jacksonVersion = '2.12.0'
-    branch = System.getenv('TRAVIS_BRANCH') != null
-        ? System.getenv('TRAVIS_BRANCH')
+    branch = System.getenv('ACTIONS_BRANCH') != null
```

Comment:
Where is `ACTIONS_BRANCH` defined? I'd recommend using the `nelonoel/branch-name@v1.0.1` action, like in `hri-mgmt-api`. Then you can call `BRANCH_NAME`.

Reply:
I replaced it with the action ✅